### PR TITLE
Create posts 2022-03-22

### PR DESCRIPTION
**Pull request by arianna-merritt**
---
name: Blog post publishing
about: 
title: 'Using task success data to improve user experience'
labels: ''
assignees: '@arianna-merritt'
---
### Preview Links
- EN: https://deploy-preview-66--blog-tbs.netlify.app/2022/03/23/task-success
- FR: https://deploy-preview-66--blogue-sct.netlify.app/2022/03/23/reussite-des-taches

### CODE REVIEW

**Review by delisma — CHANGES_REQUESTED** [review]
Everything looks good. Only the images are missing.

**Comment by arianna-merritt** [comment]
Added images. Good to go :)

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+Quel que soit votre poste, vous pouvez poser des questions sur les tendances dégagées par les résultats, ou des questions de recherche précises pour une analyse plus approfondie, ou encore examiner vous-même les commentaires.
+
+Lorsque tout le monde contribue à faire connaître les irritants et à apporter des améliorations, la culture change et devient plus axée sur le service, et les utilisateurs du contenu en bénéficient.
+
+## Obtenir une rétroaction, agir, répéter
+
+Il ne suffit pas de recueillir des commentaires. Vous devez être prêt à jouer un rôle avec ces changements nécessaires.
+
+Dans un contexte de priorités concurrentes, il peut être difficile de justifier l’abandon de certaines demandes au profit de la mise à jour d’un contenu Web déjà « approuvé » et « en ligne ».
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:113]:
```suggestion
Dans un contexte de priorités concurrentes, il peut être difficile de justifier l’abandon de certaines demandes au profit de la mise à jour d’un contenu Web déjà «&nbsp;approuvé&nbsp;» et «&nbsp;en ligne&nbsp;».

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+[Sondage sur la réussite des tâches du GC](https://conception.canada.ca/amelioration-continue/mesure/SRTGC.html)
+
+## Utiliser la rétroaction pour découvrir des problèmes
+
+> « La rétroaction des clients est critique afin de veiller à ce que les services répondent à leurs besoins et en favorisent l’amélioration continue. » – [Ligne directrice sur les services et le numérique](https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/ligne-directrice-services-numerique.html#ToC2_2)
+>
+
+Parce qu’il se déroule en continu, le sondage sur la réussite des tâches vous permettra de suivre le rendement au fil du temps. Les résultats peuvent vous aider à identifier les problèmes simples qui peuvent être résolus avec un minimum d’effort. La résolution de ces problèmes permet à votre équipe d’acquérir des compétences pour analyser la rétroaction et y donner suite.
+
+Vous découvrirez peut-être des **problèmes courants** comme ceux-ci :
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:26]:
```suggestion
Vous découvrirez peut-être des **problèmes courants** comme ceux-ci&nbsp;:

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+altLangPage: "/2022/03/23/task-success"
+date: 2022-03-23
+description: "Les données tirées du Sondage sur la réussite des tâches du GC peuvent contribuer à favoriser un changement de culture posi tif dans votre organisation."
+title: "Utiliser les données sur la réussite des tâches pour améliorer l'expérience des utilisateurs"
+---
+
+Le Secrétariat du Conseil du Trésor du Canada (SCT) et l’Éditeur principal, en collaboration avec les ministères et organismes, utilisent un **sondage sur la réussite des tâches** pour mesurer le rendement du contenu et des services Web. Les ministères peuvent employer les commentaires des utilisateurs recueillis directement par le sondage **pour améliorer le rendement du contenu des tâches principales.**
+
+Les invitations à participer au sondage sur la réussite des tâches du GC ont commencé à être publiées sur le site Canada.ca en janvier 2021. Environ 1 visiteur sur 10 est invité à contribuer des commentaires sur son expérience. S’il accepte, le sondage l’invite à :
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:10]:
```suggestion
Les invitations à participer au sondage sur la réussite des tâches du GC ont commencé à être publiées sur le site Canada.ca en janvier 2021. Environ 1 visiteur sur 10 est invité à contribuer des commentaires sur son expérience. S’il accepte, le sondage l’invite à&nbsp;:

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- identifier la tâche qu’il est venu accomplir;
+- indiquer s’il a été en mesure d’accomplir la tâche;
+- évaluer son expérience quant à la facilité d’utilisation et la satisfaction;
+- formuler un commentaire dans ses propres mots.
+
+[Sondage sur la réussite des tâches du GC](https://conception.canada.ca/amelioration-continue/mesure/SRTGC.html)
+
+## Utiliser la rétroaction pour découvrir des problèmes
+
+> « La rétroaction des clients est critique afin de veiller à ce que les services répondent à leurs besoins et en favorisent l’amélioration continue. » – [Ligne directrice sur les services et le numérique](https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/ligne-directrice-services-numerique.html#ToC2_2)
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:21]:
```suggestion
> «&nbsp;La rétroaction des clients est critique afin de veiller à ce que les services répondent à leurs besoins et en favorisent l’amélioration continue.&nbsp;» – [Ligne directrice sur les services et le numérique](https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/ligne-directrice-services-numerique.html#ToC2_2)

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+Cette façon de recueillir des commentaires permet également de mettre en évidence les **changements dans les taux de réussite**.
+
+Ces changements peuvent découler de modifications particulières apportées à votre contenu, à la politique ou même avec le grand public. Un organisation devrait être capable de mesurer  les fluctuations à un moment précis. Ceci vous permet de savoir comment résoudre les problèmes dès le début, avant qu’ils ne touchent une grande partie de votre base d’utilisateurs. Cela signifie également que vous pouvez constater des changements positifs lorsque vous corrigez un problème.
+
+À mesure que vous réglez les problèmes les plus simples et les plus faciles à gérer, examinez à nouveau les données recueillies dans le cadre du sondage. Avez-vous cessé de recevoir des messages à propos de ce problème en particulier?
+
+Pour traiter les questions de contenu plus complexes, vous aurez peut-être besoin davantage de recherche complémentaire, d’autres outils et de la collaboration avec d’autres équipes, mais les résultats du sondage vous donneront tout de même un bon point de départ.
+
+**Communiquer les réussites** aux gérants. Vous pourrez ainsi :
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:45]:
```suggestion
**Communiquer les réussites** aux gérants. Vous pourrez ainsi&nbsp;:

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+## La réussite des tâches est un sport d’équipe
+
+Promouvoir le changement au nom des utilisateurs est un sport d’équipe. Cela commence par une compréhension du problème et nécessite souvent les connaissances, les compétences de plusieurs disciplines pour le résoudre.
+
+De nombreuses équipes n’ont pas de chef de produit ou de gestionnaire de services attitré. La responsabilité de lire les commentaires des utilisateurs ou d’effectuer une analyse détaillée des résultats de sondage revient normalement à la personne responsable des rapports d’analyse Web. Ce qui peut en fait s’avérer bénéfique, car l’analyse requiert une constante attention pour repérer et surveiller les changements dans les tendances (cette responsabilité ne devrait donc pas trop souvent changer parmi les membres de l’équipe).
+
+Cependant, si une personne peut prendre la tête de la communication des résultats de sondage, tous les membres qui font partie de l’écosystème d’une tâche peuvent et doivent participer à sa correction ou à son amélioration.
+
+Il peut s’agir notamment des personnes suivantes :
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:97]:
```suggestion
Il peut s’agir notamment des personnes suivantes&nbsp;:

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+Lorsque tout le monde contribue à faire connaître les irritants et à apporter des améliorations, la culture change et devient plus axée sur le service, et les utilisateurs du contenu en bénéficient.
+
+## Obtenir une rétroaction, agir, répéter
+
+Il ne suffit pas de recueillir des commentaires. Vous devez être prêt à jouer un rôle avec ces changements nécessaires.
+
+Dans un contexte de priorités concurrentes, il peut être difficile de justifier l’abandon de certaines demandes au profit de la mise à jour d’un contenu Web déjà « approuvé » et « en ligne ».
+
+Certains pourraient être tentés à se demander : « Avec des ressources limitées, pourquoi revenir sur quelque chose qui a déjà été publié? »
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:115]:
```suggestion
Certains pourraient être tentés à se demander&nbsp;: «&nbsp;Avec des ressources limitées, pourquoi revenir sur quelque chose qui a déjà été publié?&nbsp;»

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+Publier quelque chose en ligne n’est pas la fin – c’est le début. Une fois « lancé », un cycle d’amélioration continuelle commence. La beauté du Web, c’est qu’il est facile d’apporter des améliorations à mesure que les choses changent.
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:117]:
```suggestion
Publier quelque chose en ligne n’est pas la fin – c’est le début. Une fois «&nbsp;lancé&nbsp;», un cycle d’amélioration continuelle commence. La beauté du Web, c’est qu’il est facile d’apporter des améliorations à mesure que les choses changent.

```

File: fr/_posts/2022-03-23-reussite-des-taches.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+
+Le sondage sur la réussite des tâches vous donne accès aux données et aux commentaires des personnes qui utilisent vos services. Elles ont pris le temps de vous dire qu’elles avaient échoué ou qu’elles étaient devenues frustrées. Leurs propos peuvent vous aider à présenter des arguments convaincants en faveur de l’affectation de ressources précieuses pour apporter les améliorations nécessaires.
+
+Il est important de se rappeler que l’objectif ultime de tout programme ou service n’est pas que le gouvernement parvienne à communiquer à son sujet. **L’objectif est que les gens réussissent à utiliser le programme ou le service.**
+
+## Mot de la fin
+
+Dedans l’écosystème d’une tâche, les équipes doivent travailler ensemble pour :
```
Comment (delisma) [inline fr/_posts/2022-03-23-reussite-des-taches.md:125]:
```suggestion
Dedans l’écosystème d’une tâche, les équipes doivent travailler ensemble pour&nbsp;:

```

**Review by delisma — CHANGES_REQUESTED** [review]
The French version needs a bit of revision

**Review by arianna-merritt — COMMENTED** [review]
Looks good

**Review by delisma — APPROVED** [review]
Approved

**Pull request merged**